src/utils/recommendations.ts

```diff
@@ -0,0 +1,264 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// `getAllPriorityDataClasses` contains hardly any logic and can probably do
+// with a refactoring too, given its unused arguments.
+/* c8 ignore start */
+
+// It's unclear why this function has unused parameters, but since they're
+// positional parameters, removing them risks breaking things:
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+
+type PriorityDataClassArray = {
```

Comment:
Nice types! Small nit: this doesn't look like an array, so maybe it should not be called an `-Array` :)